File: gui_tmp.py
# 🐍 RV Python — Full-Stack Example
import requests
import numpy as np

# Networking — fetch from GitHub API
response = requests.get("https://api.github.com")
print("GitHub API status:", response.status_code)

# Data Science — NumPy array analysis
scores = np.array([85, 92, 78, 90, 88])
print("Mean score:", scores.mean())

# Classic OOP
class Robot:
    def __init__(self, name):
        self.name = name
    def greet(self):
        return "Hello from " + self.name

bot = Robot("RV-Python")
print(bot.greet())

File: gui_tmp.rb
# 💎 RV Ruby — Full-Stack Example
require "net/http"

# Networking — fetch from GitHub API
response = Net::HTTP.get("https://api.github.com")
puts "GitHub response length: " + response.length.to_s

# Git status check
status = git.status()
puts "Git Status:"
puts status

# Classic OOP
class Robot
  def initialize(name)
    @name = name
  end
  def greet
    "Hello from " + @name
  end
end

bot = Robot.new("RV-Ruby")
puts bot.greet